charts/rekor/templates/server/deployment.yaml

```diff
@@ -121,3 +126,10 @@ spec:
             claimName: {{ template "rekor.server.fullname" . }}
   {{- end }}
 {{- end }}
+{{- if .Values.server.signerFileSecretOptions }}
+        - name: {{ .Values.server.signerFileSecretOptions.secretName }}-volume
+          secret:
+            secretName: {{ .Values.server.signerFileSecretOptions.secretName }}
+            items:
+            - key: {{ .Values.server.signerFileSecretOptions.privateKeySecretKey }}
```

Comment:
`path` is also required

Reply:
Good point, thanks for mentioning that detail. Added that in [e80acac](https://github.com/sigstore/helm-charts/pull/401/commits/e80acac2c4a57c237e0abda83cd92dd0f79bba61)